ctapipe/tools/dl1_merge.py

```diff
@@ -116,9 +121,9 @@ class MergeTool(Tool):
     parser.add_argument("input_files", nargs="*", type=Path)
 
     aliases = {
-        "input-dir": "MergeTool.input_dir",
-        "output": "MergeTool.output_path",
-        "pattern": "MergeTool.file_pattern",
+        "i": "MergeTool.input_dir",
+        "o": "MergeTool.output_path",
+        "p": "MergeTool.file_pattern",
```

Comment:
I wouldn't have replaced them, you added the short forms addionally

Reply:
So `-p, --pattern`, `-o, --output`, `-i, --input-dir` would be good, I think

Reply:
whups, misunderstood that